plugins/mongo_db_plugin/mongo_db_plugin.cpp

```diff
@@ -130,26 +138,38 @@ class mongo_db_plugin_impl {
 
    abi_cache_index_t abi_cache_index;
 
-   static const account_name newaccount;
-   static const account_name setabi;
+   static const action_name newaccount;
+   static const action_name setabi;
+   static const action_name updateauth;
+   static const action_name deleteauth;
+   static const permission_name owner;
+   static const permission_name active;
 
    static const std::string block_states_col;
    static const std::string blocks_col;
    static const std::string trans_col;
    static const std::string trans_traces_col;
    static const std::string actions_col;
    static const std::string accounts_col;
+   static const std::string pub_keys_col;
+   static const std::string account_controls_col;
 };
 
-const account_name mongo_db_plugin_impl::newaccount = "newaccount";
-const account_name mongo_db_plugin_impl::setabi = "setabi";
+const action_name mongo_db_plugin_impl::newaccount = N(newaccount);
```

Comment:
Suggestion: use newaccount::get_name() for actions and config::active_name and config::owner_name for these constants